<launch>
 
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <rosparam file="$(find lamp)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find lamp)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find lamp)/params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find lamp)/params/global_costmap_params.yaml" command="load" />
    
    <rosparam file="$(find lamp)/params/base_local_planner_params.yaml" command="load" />
    <rosparam file="$(find lamp)/params/move_base_params.yaml" command="load" />

		<rosparam file="$(find lamp)/params/global_planner_params.yaml" command="load" />
    
    <param name="base_global_planner" type="string" value="global_planner/GlobalPlanner" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>
    
    <remap from="odom" to="odometry/filtered" />
  </node>

</launch>
